src/components/cms/BotBuilder/BotWizard.js

```diff
@@ -303,11 +303,19 @@ class BotWizard extends React.Component {
   handleNext = () => {
     const { stepIndex } = this.state;
     const { name } = this.props;
-    this.setState({
-      stepIndex: stepIndex + 1,
-      finished: stepIndex >= 3,
-      commitMessage: 'Created Bot ' + name,
-    });
+    if (name === '') {
```

Comment:
You should strip the name before checking, otherwise cases with just multiple spaces would not run as expected.